### DISCUSSION TITLE
Alternative payment methods

### DISCUSSION THREAD

**Discussion by mholt** [community/community#10041] [Other Feature Feedback, Questions, & Ideas]
I am being asked by a potential sponsor if they can pay by ways other than credit card and PayPal. Can GitHub support ACH, wire transfer, and/or check?

**Comment by jlord** [comment]
Hiya! Right now credit card and PayPal are the only way to pay for sponsorships in our default flow. We have a [program in beta for companies](https://github.blog/changelog/2020-12-08-github-sponsors-for-companies-now-available-in-beta/) which allows them to pay for their entire sponsorship budget upfront via an invoice (which can then be paid for by ACH) so that's an option if the company is also willing to join the program (which requires a larger commitment to sponsoring open source).

**Reply by mholt** [reply]
Great! Can they already use this? Like can they already pay for a year up front via ACH?

**Reply by jlord** [reply]
Yep! What you pay upfront is for _all_ of your expected sponsorships. For instance, you decide you want to give back $50,000 to open source this year so we invoice you for that amount. Then as you make sponsorship selections it pulls from that balance rather than from a credit card and you can top up the balance (again, via invoice) as needed.

**Comment by vitoragnaldo** [comment]
Não é possível gravar nas configurações do usuário. Abra as configurações do usuário para corrigir erros/avisos e tente novamente.

como resolver este erro

using System;
using System.collections.Generic;
using System.linq;
using System.Text;
using System.Threading.tasks;

namespace Bemvindo_console
{

   class Program
   {
     static void main(string[] args)
     {

       Console.writeline("Bemvindo");
      }


    }

**Comment by bamar951** [comment]
[Readme.md](https://github.com/github/feedback/files/8654184/Readme.md)
